src/web/js/trove/image.js

```diff
@@ -242,6 +242,18 @@
     var annAngle = ann("Angle (a number 'x' where 0 <= x < 360)", image.isAngle);
     var checkAngle = p(image.isAngle, "Angle");
 
+    var canonicalizeAngle = function(angle) {
```

Comment:
That looks fine, though what's the spec for remainder on negative arguments?  (I.e. is `remainder(-4, 3)` equal to -4 because remainder doesn't make sense for negatives, -1 because only one multiple of 3 fits into -4, 1 for unknown mathematical reasons, 2 because two multiples of 3 get to the largest number less than -4, or something else?)